File: despatch.Domain/Models/Default/Response/ContactDataOrganizationResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataOrganizationResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string OrganizationId { get; set; }

        public virtual ContactDataResponse ContactData { get; set; }
        public virtual OrganizationResponse Organization { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/contact-data-organization-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { OrganizationResponse } from './organization-response';
export interface ContactDataOrganizationResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  organization?: OrganizationResponse;
  organizationId?: string;
}
